web/pgadmin/authenticate/oauth2.py

```diff
@@ -81,6 +80,7 @@ def oauth_logout():
 
         logout_user()
         if logout_url:
+            id_token = session['oauth2_token']['id_token']
```

Comment:
`id_token = session['oauth2_token']['id_token'] if 'id_token' in session['oauth2_toekn'] else None`

Reply:
Changed to `.get()` which is equivalent & more concise.